examples/user_guide/Performance_and_Debugging.ipynb

```diff
@@ -73,7 +73,43 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Layouts\n",
+    "## Reuse sessions\n",
+    "\n",
+    "In order to render a page for a user visiting your application the Panel server creates a so called session. This session has to execute the code associated with your application and then render the page template to HTML. Once the frontend has rendered the initial template it will establish a Websocket connection and render the actual Panel components into the template. This means that in order for the user to see any actual content the entire application has to be executed. While this can be sped up through the use of caching it will always take a little while for the initial page to be rendered.\n",
```

Comment:
If I'm correct, when a Panel app is served without using a Panel template (e.g. Bootstrap) there is a default template being served (from Panel or Bokeh?). That may need to be introduced.

Reply:
Agreed, was kind of skirting around this but best to explain this properly.